utils/warnings.ml

```diff
@@ -497,8 +497,14 @@ type token =
   | Num of int * int * modifier
 
 let letter_alert tokens =
-  let deprecated = function Letter (c,None) -> Some c | _ -> None in
-  let print_char ppf c =
+  let consecutive_letters (l,current) = function
+    | Letter (x, None) -> (l, x::current)
+    | _ ->
+        match current with
+        | [] | [ _ ] -> (l,current)
+        | _ :: _ :: _ -> (List.rev current :: l, [])
+  in
```

Comment:
I feel that this would be easier to read if defined just before the `fold_left`, with a comment before to explain what we are looking for.

Reply:
Done, with a bug fix along the way for "a+bc+d"